src/test/java/org/opentripplanner/middleware/docs/SwaggerTest.java

```diff
@@ -0,0 +1,54 @@
+package org.opentripplanner.middleware.docs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.middleware.OtpMiddlewareTest;
+import org.opentripplanner.middleware.utils.YamlUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.middleware.docs.PublicApiDocGenerator.getSwaggerDocs;
+import static org.opentripplanner.middleware.docs.PublicApiDocGenerator.getSwaggerDocsAsString;
+
+/**
+ * Contains test to verify that the swagger docs are generated correctly.
+ */
+public class SwaggerTest extends OtpMiddlewareTest {
+    private static final Logger LOG = LoggerFactory.getLogger(SwaggerTest.class);
+    private static final Path versionControlledSwaggerFile = new File("src/main/resources/spark-swagger-controlled-output.yaml").toPath();
+
+    /**
+     * Run this test to update the Swagger docs file located at {@link #versionControlledSwaggerFile}. This is disabled
+     * because it serves as a utility for keeping the docs file up-to-date.
+     */
+    @Test @Disabled
+    public void updateSwaggerDocs() throws IOException, InterruptedException {
+        String autoGeneratedSwagger = getSwaggerDocsAsString();
+        Files.writeString(versionControlledSwaggerFile, autoGeneratedSwagger);
+    }
```

Comment:
This test should be removed. It is providing a mechanism to perform a task, not to verify something. Furthermore, it would update something in version control.

Reply:
Sure... but where should that code go instead?

Reply:
Or @evansiroky do you think this should be a manual process?

Reply:
I think I found a workable way to do that in https://github.com/ibi-group/otp-middleware/pull/59/commits/28533eecedcccc7e6e4afae2faf33a8c5753ee77